frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card.tsx

```diff
@@ -0,0 +1,244 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ChartDonut, ChartThemeColor, ChartThemeVariant } from '@patternfly/react-charts';
+import {
+  DashboardCard,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  DashboardCardBody,
+} from '@console/internal/components/dashboard/dashboard-card';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { Dropdown, humanizeBinaryBytesWithoutB } from '@console/internal/components/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import { CapacityCardBody } from './capacity-card-body';
```

Comment:
convention is to put empty line between internal and external imports

Reply:
Acknowledged